input/examples/practitionerrole-nephrologist-darleen-mackay.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PractitionerRole xmlns="http://hl7.org/fhir">
+    <id value="nephrologist-darleen-mackay"/>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitionerrole"/>
+    </meta>
+    <practitioner>
+        <reference value="Practitioner/darleen-mackay"/>
+    </practitioner>
+    <code>
+        <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="304292004"/>
+        </coding>
+        <coding>
+            <system value="http://www.abs.gov.au/ausstats/abs@.nsf/mf/1220.0"/>
+            <code value="253322"/>
+            <display value="Renal Medicine Specialist"/>
+        </coding>
+        <text value="Nephrologist"/>
+    </code>
+    <telecom>
+        <system value="phone"/>
+        <value value="(03) 7010 8625"/>
+    </telecom>
+    <telecom>
+        <system value="email"/>
+        <value value="chau.fryer@murrabitph.example.com.au"/>
```

Comment:
wrong email replace with

Reply:
done

Reply:
still need to delete the email instance of telecom

Reply:
Yes I noticed literally 2 seconds ago that this requested email to be deleted not replaced. Done now & pushed.